### PR TITLE
orders producer generator

### PR DESCRIPTION
**Pull request by leonardiwagner**

### CODE REVIEW

File: producer/project.clj [hunk 1]
```diff
@@ -3,7 +3,10 @@
                  ;[org.clojure/core.async "0.4.500"]
                  ;[aleph "0.4.6"]
                  [ragtime "0.8.0"]
-                 ;[org.clojure/java.jdbc "0.5.8"]
+                 [com.zaxxer/HikariCP "3.4.1"]
```
[the remark this answers is not in the document]
Comment (leonardiwagner) [reply producer/project.clj]: @giseleml  can't you see this comment?

[the remark this answers is not in the document]
Comment (giseleml) [reply producer/project.clj]: I understood that a connection pool is a cache of database connections that are saved so they can be reused in future requests to the database, increasing the performance of the application. Thank you for the tip!

File: producer/project.clj [hunk 2]
```diff
@@ -3,7 +3,10 @@
...
+                 [org.clojure/java.jdbc "0.7.10"]
+                 [honeysql "0.9.8"]
```
[the remark this answers is not in the document]
Comment (giseleml) [reply producer/project.clj]: I see it uses the clojure structure. Very interesting!

File: producer/src/producer/core.clj [hunk 3]
```diff
@@ -10,14 +13,16 @@
...
-;     (dotimes [_ 3] 
-;       (async/put! consumer-channel (call-consumer) read-response))
-;     (Thread/sleep 5000)))
-
+(defn run-worker []
+  (while true
+     (do
+       (orders/insert)
+       (println "order created")
+       (Thread/sleep 1000))))
```
[the remark this answers is not in the document]
Comment (giseleml) [reply producer/src/producer/core.clj]: That will be very useful for testing, thank you

File: producer/src/producer/customers_repository.clj [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  (:require [honeysql.core :as sql]))
+
+(defn get-random-customer-id []
+  (first
+    (db/query
+      (sql/format {:select [:id]
+                   :from [:customers]
+                   :where [:= :deleted_on nil]
+                   :order-by [:%random]
+                   :limit :1}))))
```
[the remark this answers is not in the document]
Comment (giseleml) [reply producer/src/producer/customers_repository.clj]: It seems a bit tricky but I'm up to learn it! And they do look similar. Thank you for your comments!

[the remark this answers is not in the document]
Comment (leonardiwagner) [reply producer/src/producer/customers_repository.clj]: @giseleml the intent is distinct though:

- honeysql (this) is a SQL builder, you'll use your programming language to build the SELECT.. INSERT, etc.. queries, so you need to have SQL language knowledge

- sequelize (for Node.js) is an ORM (search for that as well): Object Relational Mapping, it'll translate your models (Customer, Products..) into SQL for you, like you just write `Customer.add()` and it'll run INSERT INTO customers.. automatically in background for you, you don't even need to know SQL

**Pull request merged**